### PR TITLE
#1535 Linted elements up to FoundationDropdown.jsx

### PR DESCRIPTION
**Pull request by NateBrune**

### CODE REVIEW

File: src/app/components/elements/ChangePassword.jsx [hunk 1]
```diff
@@ -212,25 +212,27 @@ class ChangePassword extends React.Component {
...
         values.confirmPassword.trim() !== newWif ? tt('g.passwords_do_not_match') : null,
-    confirmCheck: ! values.confirmCheck ? tt('g.required') : null,
-    confirmSaved: ! values.confirmSaved ? tt('g.required') : null,
+    confirmCheck: !values.confirmCheck ? tt('g.required') : null,
+    confirmSaved: !values.confirmSaved ? tt('g.required') : null,
 })
 
-import {reduxForm} from 'redux-form' // @deprecated, instead use: app/utils/ReactForm.js
+import {reduxForm} from 'redux-form'
+ // @deprecated, instead use: app/utils/ReactForm.js
```
Comment (goldibex) [inline src/app/components/elements/ChangePassword.jsx]: i'd actually move this comment back

File: src/app/components/elements/ChangePassword.jsx [hunk 2]
```diff
@@ -256,13 +258,16 @@ export default reduxForm(
                 twofa,
                 // signingKey provides the password if it was not provided in auths
                 signingKey: authType ? password : null,
-                accountName, auths,
-                onSuccess: success, onError: error,
+                accountName,
+auths,
+                onSuccess: success,
+onError: error,
                 // notifySuccess: 'Change password success'
```
Comment (goldibex) [inline src/app/components/elements/ChangePassword.jsx]: can you fix the spacing here

File: src/app/components/elements/ChangePassword.jsx [hunk 3]
```diff
@@ -256,13 +258,16 @@ export default reduxForm(
...
             }))
         },
         notify: (message) => {
-            dispatch({type: 'ADD_NOTIFICATION', payload: {
+            dispatch({type: 'ADD_NOTIFICATION',
```
Comment (goldibex) [inline src/app/components/elements/ChangePassword.jsx]: weird indentation here

File: src/app/components/elements/DepthChart.jsx [hunk 4]
```diff
@@ -124,45 +122,52 @@ function getMinMax(bids, asks) {
...
+    const series = [];
 
     const {min, max} = getMinMax(bids, asks);
 
     if(process.env.BROWSER) {
         if(bids[0]) {
-            series.push({step: 'right', name: tt('g.bid'), color: 'rgba(0,150,0,1.0)', fillColor: 'rgba(0,150,0,0.2)', tooltip: {valueSuffix: ' ' + LIQUID_TICKER},
-             data:  bids})
+            series.push({step: 'right',
+name: tt('g.bid'),
```
Comment (goldibex) [inline src/app/components/elements/DepthChart.jsx]: weird indentation, again.

**Comment by bnchdrff** [comment]
this will be addressed by #2081

**Pull request closed without merging**